Task/Task-NonWGSRareDiseaseTestOrderCancelled-MarkInError-Example.json

```diff
@@ -0,0 +1,47 @@
+{
+  "resourceType": "Task",
+  "id": "Task-NonWGSRareDiseaseTestOrderCancelled-MarkInError-Example",
+  "basedOn": [
+    {
+      "reference": "ServiceRequest/ServiceRequest-NonWGSTestOrderForm-MarkInError-Example"
+    }
+  ],
+  "status": "entered-in-error",
+  "statusReason": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/task-statusReason-genomics",
```

Comment:
This doesn't directly match the cancellation reason listed in the scenario, could you add the list used in the dropdown on page 5 to the statusReason codesystem and valueset and use the exact code?